### ISSUE TITLE
Bug Report / Feature Request: Polylang Support

### ISSUE THREAD

**Issue by dkd-talwar** [stcr/subscribe-to-comments-reloaded#203]
Here is the scenario:
1) A website, that is using Polylang to manage the multilingual feature with two languages ( en_GB and de_DE) and by default language is de_DE , i.e. German site opens on http://example.com and English site opens on http://example.com/de

2) By default page to manage comments subscriptions is http://example.com/comment-subscriptions/
However, if http://example.com/en/comment-subscriptions/ page is opened, then content on the page comes in English because of PO and MO files within text-domain /langs

3) Now the problem is: Under comment box on any blog post, if any comment is made with comments-subscription, then there will be this text under comments are,
"You have subscribed to this article. You can edit your comment subscriptions here: 
My comment subscriptions."
with "My comment subscriptions" hyperlinked with http://example.com/comment-subscriptions/ on english page even. However, it should be http://example.com/en/comment-subscriptions/  on English blog post and 
http://example.com/comment-subscriptions/  on German blog post. 
Similarly with the mails it should be, user subscribing comments from English blog post should get an email in English.

Kindly guide and also Kindly provide polylang support.

**Comment by wickedpuppy** [comment]
Same problem here.

**Comment by Reedyseth** [comment]
Hi Guys, I apologize for the late response I will be trying to replicate the issue. 

Regards.!!

**Comment by wickedpuppy** [comment]
Great! Looking forward to that!

**Comment by Reedyseth** [comment]
> 3) Now the problem is: Under comment box on any blog post, if any comment is made with comments-subscription, then there will be this text under comments are,
> "You have subscribed to this article. You can edit your comment subscriptions here: 
> My comment subscriptions."
> with "My comment subscriptions" hyperlinked with http://example.com/comment-subscriptions/ on english page even. However, it should be http://example.com/en/comment-subscriptions/ on English blog post and 
> http://example.com/comment-subscriptions/ on German blog post. 
> Similarly with the mails it should be, user subscribing comments from English blog post should get an email in English.

@dkd-talwar @wickedpuppy 

So on this case you only need the prefix (en) before the comment subscription slug?

**Comment by wickedpuppy** [comment]
What I'm developing now defaults to english, and has a prefix for the translations only,

Similarly, there's also the issue with the language used in the feedback e-mails.

**Comment by dkd-talwar** [comment]
This is How handled the issue temporarily, Still there are bugs left.
a) I have created custom page template, as you have defined in
https://github.com/stcr/subscribe-to-comments-reloaded/wiki/KB#create-a-real-management-page
b) Then created 2 pages for both languages selected the page template,
c) Then used the code in Pull request #208 (Certainly not perfect, and require optimization)

I am now able to switch between pages on both languages, but left with certain bugs,
I have described them here,
https://github.com/stcr/subscribe-to-comments-reloaded/pull/208#issue-141021221

**Comment by dkd-talwar** [comment]
Yep., Issue with emails are also there. Fixed partially with Po and Mo file, But still not completely fixed.

**Comment by Reedyseth** [comment]
> I am now able to switch between pages on both languages, but left with certain bugs,
> I have described them here,
> #208 (comment)

OK As soon as I have a change to test the https://github.com/stcr/subscribe-to-comments-reloaded/pull/208 I update my findings.

**Comment by dkd-talwar** [comment]
Using the same setup as mentioned in 
https://github.com/stcr/subscribe-to-comments-reloaded/issues/203#issuecomment-225612085

One question
Subscribe_reloaded_management_content -> is for content in email right ? To manage the subscriptions ?
but this text is visible after email is entered on /comments-subscriptions/ page by a non-author
I think there should be a another option in stcr_manager.php

Or am I missing something
Is there any existing option for the text displayed after the email entered on /comments-subscriptions/ page .
Something like, "An email is sent to you please check your email"

**Comment by wickedpuppy** [comment]
Any news?
